postpic/helper.py

```diff
@@ -487,18 +487,26 @@ def omega_yee(kmesh):
     return omega_yee
 
 
+def omega_free(mesh):
+    k2 = sum(ki**2 for ki in mesh)
+    return PhysicalConstants.c * np.sqrt(k2)
+
+
 def _kspace_helper_cutfields(component, fields, extent):
     slices = fields[component]._extent_to_slices(extent)
     return {k: f[slices] for k, f in fields.items()}
 
 
-def kspace_epoch_like(component, fields, extent=None, omega_func=None, align_to='B'):
+def kspace_epoch_like_old(component, fields, extent=None, omega_func=omega_free, align_to='B'):
```

Comment:
As this is only for comparison please mark it as such. There is a deprecation decorator in the helper.py